app/controllers/admin/works_controller.rb

```diff
@@ -96,6 +97,18 @@ def submit_ohms_xml
     end
   end
 
+  # PATCH/PUT /admin/works/ab2323ac/remove_ohms_xml
+  def remove_ohms_xml
+    @work.oral_history_content!.update!(ohms_xml_text: nil)
+    redirect_to admin_work_path(@work, anchor: "nav-oral-histories"), notice: "OHMS XML file removed."
+  end
+
+  # GET /admin/works/ab2323ac/download_ohms_xml
+  def download_ohms_xml
+    send_data @work.oral_history_content!.ohms_xml_text,
+      :type => 'text/xml; charset=UTF-8;',
+      :disposition => "attachment; filename=#{@work.oral_history_content!.ohms_xml.accession}.xml"
```

Comment:
Rather than construct the header manually, we have a gem we use in other places, that helps avoid errors. Like special encoding needed if there's unicode in the filename, which there shouldn't be in this case, but just DRY up to use the utility is safer...

```ruby
:disposition => ContentDisposition.format(disposition: "attachment", filename: "#{@work.oral_history_content!.ohms_xml.accession}.xml")
```

Already have this gem available and used in our project. https://github.com/shrinerb/content_disposition